packages/orbit-components/src/Alert/index.tsx

```diff
@@ -290,37 +128,56 @@ const Alert = (props: Props) => {
     labelClose,
   } = props;
   return (
-    <StyledAlert
-      type={type}
-      icon={icon}
+    <div
+      className={cx(
+        "rounded-large text-ink-dark font-base text-normal p-sm relative box-border flex w-full border border-t-[3px] leading-normal",
```

Comment:
do we maybe want a token for `border-t-[3px]`?

Reply:
As agreed on DMs, it's currently not justifiable to have a token here. If the usage of a 3px border evolves, we will reconsider